images/scripts/fedora-43-anaconda-payload.bootstrap

```diff
@@ -3,4 +3,9 @@ set -eux
 
 OUTPUT="$1"
 
-PYTHONPATH=. python3 images/scripts/create-anaconda-payload --output=$OUTPUT --image fedora-43
+WORKDIR=$(mktemp -d)
+PYTHONPATH=. python3 images/scripts/create-anaconda-liveimg-payload --output="$WORKDIR" --image fedora-43
+PYTHONPATH=. python3 images/scripts/create-anaconda-dnf-payload --output="$WORKDIR" --image fedora-43
```

Comment:
Not entirely sure how this works, does it set any environment variables here? If so they might collide on some variable names given the two scripts have similar functionality

Reply:
Hm, not sure what do you mean? Which variable do you have in mind as problematic?

Reply:
No variables are problematic right now but I think the behavior makes it fragile. Can easily create a variable in one that exists in the other, which would be overridden or override the other if one is not careful of the variable naming